UnitsNet/UnitDefinitions/ThermalResistance.json

```diff
@@ -2,6 +2,11 @@
   "Name": "ThermalResistance",
   "BaseUnit": "SquareMeterKelvinPerKilowatt",
```

Comment:
Not the scope of this PR, but it seems this is maybe not the correct base unit, but rather `KelvinPerWatt` `K/W`.
http://www.wolframalpha.com/input/?i=thermal+resistance

Reply:
Indeed!